src/ui/src/builder/BuilderFieldsTools.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+	<div
+		class="BuilderFieldsTools colorTransformer"
+		:data-automation-key="props.fieldKey"
+	>
+		<div class="tools">
+			<div
+				v-for="(tool, toolName) in tools"
+				:key="toolName"
+				class="tool"
+				@click="editTool(toolName)"
+			>
+				<div class="toolName">{{ toolName }}</div>
+				<button class="delete" @click.stop="deleteTool(toolName)">
+					<i class="material-symbols-outlined">delete</i>
+				</button>
+			</div>
+		</div>
+
+		<WdsButton
+			variant="builder"
+			size="small"
+			@click="resetAndShowToolFormModal"
+		>
+			<i class="material-symbols-outlined">add</i>
+			Add tool</WdsButton
+		>
```

Comment:
Could we create a task to have `icon` attribute in `WdsButton` ? For example, here you should have a `gap` between the icon and the test [Figma](https://www.figma.com/design/jgLDtwVwg3hReC1t4Vw20D/.WDS-Writer-Design-System?node-id=666-2324&t=DubUJE0j7EF9N4De-4)

Reply:
It does have a gap, the `WdsButton` is actually a flex with a gap in it, so whatever you put in the slot comes with a gap.